pmm/pmm2-release.groovy

```diff
@@ -485,15 +486,17 @@ ENDSSH
                         aws s3 cp --only-show-errors s3://percona-vm/pmm-client-\${VERSION}.docker pmm-client-\${VERSION}.docker
                     """
                 }
-                sh """
-                    ssh -o StrictHostKeyChecking=no -i ${KEY_PATH} ${USER}@repo.ci.percona.com << 'ENDSSH'
-                        ssh -i ~/.ssh/id_rsa_downloads -p 2222 jenkins@jenkins-deploy.jenkins-deploy.web.r.int.percona.com "mkdir -p /data/downloads/pmm2/\${VERSION}/docker || true"
-                        sha256sum pmm-server-\${VERSION}.docker > pmm-server-\${VERSION}.sha256sum
-                        sha256sum pmm-client-\${VERSION}.docker > pmm-client-\${VERSION}.sha256sum
-                        scp -i ~/.ssh/id_rsa_downloads -P 2222 pmm-server-\${VERSION}.docker pmm-server-\${VERSION}.sha256sum jenkins@jenkins-deploy.jenkins-deploy.web.r.int.percona.com:/data/downloads/pmm2/\${VERSION}/docker/
-                        scp -i ~/.ssh/id_rsa_downloads -P 2222 pmm-client-\${VERSION}.docker pmm-client-\${VERSION}.sha256sum jenkins@jenkins-deploy.jenkins-deploy.web.r.int.percona.com:/data/downloads/pmm2/\${VERSION}/docker/
+                withCredentials([sshUserPrivateKey(credentialsId: 'repo.ci.percona.com', keyFileVariable: 'KEY_PATH', usernameVariable: 'USER')]) {
+                    sh """
+                        ssh -o StrictHostKeyChecking=no -i ${KEY_PATH} ${USER}@repo.ci.percona.com << 'ENDSSH'
+                        ssh -p 2222 -o StrictHostKeyChecking=no ${REPO_HOST} "mkdir -p /data/downloads/pmm2/${VERSION}/docker"
+                        sha256sum pmm-server-${VERSION}.docker > pmm-server-${VERSION}.sha256sum
+                        sha256sum pmm-client-${VERSION}.docker > pmm-client-${VERSION}.sha256sum
+                        scp -P 2222 -o ConnectTimeout=1 -o StrictHostKeyChecking=no pmm-server-${VERSION}.docker pmm-server-${VERSION}.sha256sum ${REPO_HOST}:/data/downloads/pmm2/${VERSION}/docker/
+                        scp -P 2222 -o ConnectTimeout=1 -o StrictHostKeyChecking=no pmm-client-${VERSION}.docker pmm-client-${VERSION}.sha256sum ${REPO_HOST}:/data/downloads/pmm2/${VERSION}/docker/
```

Comment:
It will not work.
You are executing `aws cp` command on jenkins slave and put dockers and ova files on jenkins slave
then you are doing ssh to a repo server and trying to calculate sha256 on it. but there is no such files on it.